packages/table/src/interactions/selectable.tsx

```diff
@@ -84,11 +92,17 @@ export interface IDragSelectableProps extends ISelectableProps {
      * coordinate data representing a drag. If no valid region can be found,
      * `null` may be returned.
      */
-    locateDrag: (event: MouseEvent, coords: ICoordinateData) => IRegion;
+    locateDrag: (event: MouseEvent, coords: ICoordinateData, returnEndOnly?: boolean) => IRegion;
```

Comment:
Kind of a weird parameter, but seemed necessary:
- If `returnEndOnly=false`, then `locateDrag` returns the region that stretches the activation cell to the most recent target cell.
- If `returnEndOnly=true`, then `locateDrag` returns the most recent target cell by itself, so we can join it with the `focusedCell` using our own logic.